src/common/rollbar-api.js

```diff
@@ -36,18 +35,33 @@ class RollbarAPI {
     return this.processResponse(resp);
   }
 
+  async sigendURLsourcemaps(request) {
+
+    const json = JSON.stringify({ version: request.version , prefix_url: request.baseUrl});
+
+    const resp = await this.axios.post(
+      '/signed_url/sourcemaps', json,
+        {headers: {
+            // axios needs some help with headers for form data.
```

Comment:
This comment is out of place, as the request here doesn't use form data or its headers.